openrtb_ext/imp_smilewanted.go

```diff
@@ -0,0 +1,6 @@
+package openrtb_ext
+
+// ExtImpSmileWanted defines the contract for bidrequest.imp[i].ext.smilewanted
```

Comment:
Since your adapter code doesn't really. unmarshal the imp ext and reads/modifies the `ZoneID`, you don't need this file and it can be safely removed

Reply:
Done